parsec/backend/postgresql/user_queries/find.py

```diff
@@ -88,6 +102,23 @@ async def query_find(
     return results, len(all_results)
 
 
+@query()
+async def query_retrieve_active_human_by_email(
+    conn, organization_id: OrganizationID, email: str
+) -> Optional[HumanFindResultItem]:
```

Comment:
There could be multiple user with the same human handle (but at most one which is not revoked).

Given the query currently doesn't filter on the revoked field (but only return a boolean is_revoked), I guess we can have trouble in case we have multiple user with the same human handle.

This should be solved by the changes I proposed on the query (i.e. filter on revoked and so return only zero on one row) 😃